src/pages/SearchPage.js

```diff
@@ -32,82 +31,73 @@ const propTypes = {
 
     /** All reports shared with the user */
     reports: PropTypes.objectOf(reportPropTypes),
-
-    /** Window Dimensions Props */
-    ...windowDimensionsPropTypes,
-
-    ...withLocalizePropTypes,
 };
 
-const defaultProps = {
-    betas: [],
-    personalDetails: {},
-    reports: {},
-};
+function SearchPage({betas = [], personalDetails = {}, reports = {}}) {
+    // Data for initialization (runs only on the first render)
+    const {recentReports: initialRecentReports, personalDetails: initialPersonalDetails, userToInvite: initialUserToInvite} = useMemo(() => OptionsListUtils.getSearchOptions(reports, personalDetails, '', betas), []);
```

Comment:
Linter will complain on this, so just disable this rule
```suggestion
    // eslint-disable-next-line react-hooks/exhaustive-deps
    const {recentReports: initialRecentReports, personalDetails: initialPersonalDetails, userToInvite: initialUserToInvite} = useMemo(() => OptionsListUtils.getSearchOptions(reports, personalDetails, '', betas), []);
```